samples/sensor/light_polling/sample.yaml

```diff
@@ -33,9 +33,11 @@ tests:
     platform_allow:
       - frdm_mcxw71/mcxw716c
       - adafruit_qt_py_rp2040/rp2040       # adafruit_tsl2591 shield
```

Comment:
Add as integration platform as well so that this one test can serve as the "main" test for this shield in CI (including here in this PR since as it is right now it hasn't run as part of the checks)

Reply:
Done